Sources/Tracking/CustomerIO.swift

```diff
@@ -115,6 +136,15 @@ public class CustomerIO: CustomerIOInstance {
     internal var implementation: CustomerIOImplementation?
 
     internal var globalData: GlobalDataStore = CioGlobalDataStore()
+    
+    public var autoScreenViewBody: () -> ScreenViewData {
```

Comment:
I imagined this being in the `CustomerIO.config {}` code, only. Is there a reason for there being a function here?

Reply:
yeah so this is to allow people to hook in and gather a custom body instead of our default (which today is nothing). Moving around a bit to help clarify